### PR TITLE
1.8/1439 improve usability of search screen

### PR DESCRIPTION
**Pull request by Nikomus**
https://trello.com/c/KtisvGVj/1439-improve-usability-of-search-screen-vv-ee

### CODE REVIEW

**Comment by AlinChindea** [comment]
There are a couple of tests missing but am sure you noticed that :-). Otherwise, looks good.

File: tests/Feature/Store/SearchPageTest.php [hunk 1]
```diff
@@ -363,6 +363,30 @@ public function itCanPaginateWithNumberedPages()
         ;
     }
 
+    /** @test */
+    public function itShowsCentreLabelsForUsersByDefault()
+    {
+        $this->markTestIncomplete(
```
Comment (charlesstrange2) [inline tests/Feature/Store/SearchPageTest.php]: Sad but unavoidable until i get the selective mysql testing sorted.

Comment (Nikomus) [reply tests/Feature/Store/SearchPageTest.php]: This is just one I haven't filled in yet.

**Comment by AlinChindea** [comment]
I think there's enough duplication in the tests to warrant a `setUp()` method. your centre, centre user, maybe registrations, too. but don't know if you have time to refactor.

**Pull request merged**